### DISCUSSION TITLE
Can't find integration?

### DISCUSSION THREAD

**Discussion by simongis** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#293] [Q&A] [answered]
Instructions mention that I should be able to search and install from the HACS GUI.  I am only on my 1st day of HACS, but if I search for the integration, I can not spot it.

Before I go to manual install approach, have I missed somehting? 

![image](https://user-images.githubusercontent.com/2769383/170800722-45aabbbd-1df6-4155-b809-bd19b3b2261d.png)

Home Assistant Core 2022.5.5
Home Assistant Supervisor 2022.05.3
Home Assistant OS 8.1

**Comment by PiotrMachowski** [comment, marked as answer]
This screen is not HACS, this is built-in HA screen for setting up an integration

**Reply by simongis** [reply]
ah! ok. time to go watch some more youtube videos and googling.  Thanks for the pointer.  

EDIT - just realised the difference.  Thank you!
